### ISSUE TITLE
pyGimli through windows anaconda (using conda install)

### ISSUE THREAD

**Issue by Coastal0** [gimli-org/gimli#85]
Hi all,

I see there are options for installing pygimli through linux-based anaconda, as well as windows based pygimli distributions, but no option for pygimli through windows-based anaconda?

Is there a way to install gimli through conda that I've missed?

**Comment by halbmy** [comment]
No you did not miss anything. It is on the list. Facing issues like #83 I would rather opt for MiniConda instead of Anaconda but I understand that it is way easier for typical Windows users.

We can let this open as a reminder to provide conda packages for Windows although I think that the MSI and wheel installers should work well with Anaconda.

**Comment by Coastal0** [comment]
For anyone in a similar place, the 'WinPython' install (with the wheel) is very easy.

**Comment by GGDRriedel** [comment]
create an anaconda environment like 
`conda create --name pygimli python=3.6`

activate it
download the wheel
install it via pip


Anaconda automatically detects that you are pip'ing in an env 

You have to install the dependencies by hand. 

So far this is matplotlib, numpy and qt

**Comment by halbmy** [comment]
We have now conda versions for Windows, however we are still testing. Version 1.0.11.1 is available for Python 3.6, and there's also a current version 1.0.12 for Python 3.7 (recommended). They can be installed by
```bash
conda create -n blabla -c gimli pygimli  # optionally add pygimli=1.0.11 or python=3.6 etc.
```
Please check it out and report back. We are about finalizing the 1.0.x releases and will then starting to release 1.1 versions.
